_posts/2023-02-22-user-defined-operators.md

```diff
@@ -0,0 +1,598 @@
+---
+layout: post
+published: true
+title: 'Feature Deep-Dive: User-Defined Operators'
+date: '2023-02-22'
+author: Kamil Śliwak, Matheus Aguiar
+category: Explainers
+---
+
+The prominent new feature in [Solidity 0.8.19](https://github.com/ethereum/solidity/releases/tag/v0.8.19)
+is the support for defining *operators* on [user-defined value types](https://docs.soliditylang.org/en/latest/types.html#user-defined-value-types) (UDVTs).
```

Comment:
```suggestion
is support for defining *operators* on [user-defined value types](https://docs.soliditylang.org/en/latest/types.html#user-defined-value-types) (UDVTs).
```

Reply:
The article "the" is correct. 🤔

Reply:
for me it sounded weird but you might be right here 👍